vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Vagrant
+
+Here you will find a `Vagrantfile` to build a development environment for `kvm-vmi`.
+
+# Requirements
+
+- `vagrant`
+- `vagrant-libvirt` plugin
+- `ansible >= 2.2.1.0` (use a virtualenv, install it via pip)
```

Comment:
Then I'd opt virtualenv out. Like this makes wonder "is virtualenv necessary or not?"